### ISSUE TITLE
Continuous zooming in or out with touchpad on Leaflet map

### ISSUE THREAD

**Issue by BabyRamyaRaavi** [Leaflet/Leaflet#9508]
### Checklist

- [x] I've looked at the [documentation](https://leafletjs.com/reference.html) to make sure the behavior isn't documented and expected.
- [x] I'm sure this is an issue with Leaflet, not with my app or other dependencies (Angular, Cordova, React, etc.).
- [x] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this hasn't been reported yet.
- [x] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Steps to reproduce

Description: When using a touchpad to zoom in or out on the Leaflet map, a single zoom action results in multiple zoom operations. This causes the map to zoom excessively, making it difficult to control the zoom level accurately.

Open the Leaflet map.
Use a touchpad to perform a zoom-in gesture.
Observe the map continuously zooming in multiple times for a single gesture.

### Expected behavior

The map should zoom in or out only once per touchpad gesture

### Current behavior

The map zooms in or out multiple times for a single touchpad gesture.

### Minimal example reproducing the issue

https://apteanonline-my.sharepoint.com/:v:/g/personal/braavi_aptean_com/EeltDQHFdMxIupXx3qGcZToBcbK47UBsHSOdlV80i7tYBg

### Environment

Leaflet version: 1.9.3
Browser (with version): The issue is there in all Browsers

**Comment by IvanSanchez** [comment]
Duplicate of https://github.com/Leaflet/Leaflet/issues/4696
